Sources/Sentry/SentryTime.mm

```diff
@@ -6,6 +6,15 @@
 
 #import "SentryMachLogging.hpp"
 
+uint64_t
+timeIntervalToNanoseconds(double seconds)
```

Comment:
Unfortunately can't do it, because `Foundation` needs to be imported, and then for some reason that breaks a bunch of weird stuff, the compiler won't find things like NSString/NSObject. I wasn't able to crack it 😕 It's weird because it's already importing foundation in the implementation file, but adding that to the header takes it all down ¯\\_(ツ)_/¯